starter-code/index.html

```diff
@@ -1,31 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Instagram</title>
+ <title>Instagram</title>
+ <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+ 
+ 
+ 
+ <div class="container">
+ <header>
+   <img src="./images/phones.png" alt="Iphone">
+ </header>
 
-  Capture and Share the World's Moments
+ <section>
+   <nav>
+     <img src="./images/brand.png" alt="Logo instagram">
```

Comment:
Same here, the brand logo and the log in buttons could probably be defined as a `<header>`, more than a `<nav>`.
`<nav>` usually contains links to navigate through the website.